src/DETHRACE/common/brucetrk.c

```diff
@@ -14,10 +14,14 @@
 #include <stdlib.h>
 #include <string.h>
 
+// GLOBAL: CARM95 0x53e488
```

Comment:
Looks good. My only nitpick is I would have formatted the addresses as `0x%08x` instead of `0x%x`